peps/pep-0810.rst

```diff
@@ -324,12 +324,16 @@ Example with ``lazy from ... import``:
   # Accessing 'loads' now reifies it (json already loaded, no re-import)
   data = loads(result)
 
-A module may contain a :data:`!__lazy_modules__` attribute, which is a
-sequence of fully qualified module names (strings) to make *potentially lazy*
-(as if the ``lazy`` keyword was used). This attribute is checked on each
-``import`` statement to determine whether the import should be made
-*potentially lazy*. When a module is made lazy this way, from-imports using
-that module are also lazy, but not necessarily imports of sub-modules.
+A module may define a :data:`!__lazy_modules__` variable in its global scope,
+which specifies which module names should be made *potentially lazy* (as if the
+``lazy`` keyword was used). This variable is checked on each ``import``
+statement to determine whether the import should be made *potentially lazy*.
+The check is performed by calling ``__contains__`` on the
+:data:`!__lazy_modules__` object with a string containing the fully qualified
+module name being imported. Typically, :data:`!__lazy_modules__` is a list of
```

Comment:
Sounds like it should be a set instead? Of course both would work, but a set is more idiomatic for membership testing.

Reply:
👍